quickstarts/microsoft.storage/storage-static-website/scripts/.gitattributes

```diff
@@ -0,0 +1 @@
+*.ps1 text eol=lf
```

Comment:
why is this needed?

Reply:
Is there a bicep issue for this?

Reply:
Hard to argue that it's a bicep issue.  git by default automatically changes the file newlines based on the OS, Bicep just sucks in what git downloaded, without making any changes.

You could use replace() to standardize them, but since this is not an actual deployment issue but rather a diff against the generated JSON, it shouldn't happen to normal useers.